### DISCUSSION TITLE
RMarkdown

### DISCUSSION THREAD

**Discussion by ap1987-ap** [munoztd0/DAP_2021#15] [Q&A] [answered]
Just a general question,

regarding RMarkdown..is it ok to use RNotebook too, or HAS it to be RMarkdown?

Thank you
Anja

**Comment by munoztd0** [comment, marked as answer]
It can be a Notebook, it should just still be reproducible in other machines.

**Comment by ap1987-ap** [comment]
Ok, thank you.On 21 Dec 2021 11:32, David Muñoz Tord ***@***.***> wrote:
It can be a Notebook, it should just still be reproducible in other machines.
